model/resource/cloud_provider/gcp/project.yaml

```diff
@@ -0,0 +1,12 @@
+groups:
+  - id: gcp.project
+    prefix: gcp.project
+    type: resource
+    stability: experimental
+    brief: >
+      Project related attributes.
+    attributes:
+      - ref: gcp.project.id
```

Comment:
I think this file can be removed: `gcp.project.id` is not used as a *resource* attribute in messaging semconv